src/esp/metadata/managers/LightLayoutAttributesManager.cpp

```diff
@@ -140,7 +140,7 @@ void LightLayoutAttributesManager::setLightInstanceValsFromJSONDoc(
 
 LightLayoutAttributes::ptr LightLayoutAttributesManager::initNewObjectInternal(
     const std::string& handleName,
-    bool builtFromConfig) {
+    CORRADE_UNUSED bool builtFromConfig) {
```

Comment:
Same here and in other `cpp` files and as @jturner65 says, just drop the name altogether